src/coreclr/src/inc/corinfo.h

```diff
@@ -891,28 +891,6 @@ enum CorInfoException
 
 enum CorInfoIntrinsics
 {
```

Comment:
This also needs a new GUID for JITEEVersionIdentifier.

Reply:
Updated the GUID.
(direct copy and paste `uuidgen.exe -s` comamnd output, which does not have spaces before `}` and after `{` on the fourth line, VS2019)